build/target-repository/Dockerfile

```diff
@@ -31,23 +31,11 @@ RUN apt-get update && apt-get install -y \
 COPY --from=composer:2.0.9 /usr/bin/composer /usr/bin/composer
 ENV COMPOSER_ALLOW_SUPERUSER=1 COMPOSER_MEMORY_LIMIT=-1 COMPOSER_NO_INTERACTION=1
 
-# Run php-scoper, results go to /scoped
-RUN wget https://github.com/humbug/php-scoper/releases/download/0.14.0/php-scoper.phar -N --no-verbose
-
 # This is to make parsing version possible
 COPY .git .git
 
-# First copy composer.json only to leverage the build cache (as long as not git-committing)
-COPY composer.json composer.json
-RUN composer install --no-dev --ansi
-
-# Add source and generate full autoloader
```

Comment:
Composer is already dumped in prefixed version, so dump would override and break it